iml-wasm-components/iml-stratagem/src/lib.rs

```diff
@@ -248,41 +173,31 @@ pub fn update(msg: Msg, model: &mut Model, orders: &mut impl Orders<Msg>) {
 
             model.validate();
 
-            if model.config_valid() {
-                model.enable_stratagem_button = model.create_enable_stratagem_model();
+            model.enable_stratagem_button = if model.config_valid() {
```

Comment:
What do you think about having `model.create_enable_stratagem_model()` return an option?